src/libs/actions/Timing.ts

```diff
@@ -4,15 +4,20 @@ import Firebase from '../Firebase';
 import * as API from '../API';
 import Log from '../Log';
 
-let timestampData = {};
+type TimestampData = {
+    startTime: number;
+    shouldUseFirebase: boolean;
+};
+
+let timestampData: Record<string, TimestampData> = {};
 
 /**
  * Start a performance timing measurement
  *
- * @param {String} eventName
- * @param {Boolean} shouldUseFirebase - adds an additional trace in Firebase
+ * @param eventName
+ * @param shouldUseFirebase - adds an additional trace in Firebase
  */
-function start(eventName, shouldUseFirebase = false) {
+function start(eventName: string, shouldUseFirebase = false) {
```

Comment:
Why aren't we adding the type for the boolean params in this file?

```suggestion
function start(eventName: string, shouldUseFirebase: boolean = false) {
```